mitiq/benchmarks/tests/test_randomized_clifford_t_circuit.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (C) Unitary Fund
+#
+# This source code is licensed under the GPL license (v3) found in the
+# LICENSE file in the root directory of this source tree.
+
+"""Tests for quantum volume circuits. The Cirq functions that do the main work
+are tested here:
+cirq-core/cirq/contrib/quantum_volume/quantum_volume_test.py
+
+Tests below check that generate_quantum_volume_circuit() works as a wrapper and
+fits with Mitiq's interface.
+"""
+
+import pytest
+
+from mitiq import SUPPORTED_PROGRAM_TYPES
+from mitiq.benchmarks.randomized_clifford_t_circuit import (
+    generate_random_clifford_t_circuit,
+)
+
+
+def test_generate_model_circuit_with_seed():
```

Comment:
I think this test name is copied from another file, and I know we use it in other places, but I think we can make it a little more descriptive of what the actual intention is. Maybe something like `test_seed_circuit_equality`? I don't love that name either, but I'm sure we can come up something better than "model circuit".

Reply:
test_seed_circuit_equality sounds good to me. we can change it if we find a better name :)